content/docs/handling-events.md

```diff
@@ -1,43 +1,43 @@
 ---
 id: handling-events
-title: Handling Events
+title: Bắt Sự kiện
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Việc bắt sự kiện của những element React rất giống với những element DOM. Có một số khác biệt về cú pháp như:
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
+* Những sự kiện của React được đặt tên theo dạng camelCase, thay vì lowercase.
+* Với JSX, bạn có thể sử dụng hàm để bắt sự kiện thay vì phải truyền vào một chuỗi.
```

Comment:
It should be: Với JSX, bạn truyền vào một "hàm" (function) để bắt sự kiện thay vì truyền vào một chuỗi.